src/provider/store/Store.ts

```diff
@@ -39,15 +39,18 @@ export class Store<S, A extends Action<S>> extends AsyncInit implements StoreAPI
         this._currentState = initialState;
     }
 
+    public get api(): StoreAPI<S, A> {
+        return {
+            state: this.state,
```

Comment:
Changing to `state: this.getState()` still wouldn't fix the issue. `StoreAPI.state` needs to be a getter, but it's a static property. If you pass `this._store.api` down into a component, the state that that component sees is never going to update.

I can see the reasoning in having a trimmed-down type that we pass into React - but we could do that by keeping `Actionable` defined as `storeApi: StoreAPI<RootState, RootAction>;`, but then passing `this._store` (since the `Store` class also implements `StoreAPI`) rather than `this._store.api`.

Hopefully I'm explaining that properly... If not, let's discuss.

Reply:
Switched passing the store down and `StoreAPI` is now just: 
```ts
export type StoreAPI<S> = Pick<Store<S>, 'dispatch' | 'state'>;
```